File: HCSZapato/HCSZapato/MENU.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace HCSZapato
{
    public partial class MENU : Form
    {
        public MENU()
        {
            InitializeComponent();
        }

        private void zapatoToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Zapato1 Z1 = new Zapato1();
            Z1.Show();
        }

        private void deportivoToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Deportivo1 D1 = new Deportivo1();
            D1.Show();
        }

        private void casualToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Casual1 C1 = new Casual1();
            C1.Show();
        }

        private void salirToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Dispose();
        }

    }
}

File: WindowsApplication1/WindowsApplication1/MENU.vb
Public Class MENU

    Private Sub ZapatoToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles ZapatoToolStripMenuItem.Click
        Zapato1.MdiParent = Me
        Zapato1.Show()
    End Sub

    Private Sub DeportivoToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles DeportivoToolStripMenuItem.Click
        Deportivo1.MdiParent = Me
        Deportivo1.Show()
    End Sub

    Private Sub CasualToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles CasualToolStripMenuItem.Click
        Casual1.MdiParent = Me
        Casual1.Show()
    End Sub

    Private Sub SalirToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles SalirToolStripMenuItem.Click
        Dispose()
    End Sub

    Private Sub InicioToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles InicioToolStripMenuItem.Click

    End Sub
End Class
